### PR TITLE
Expose is approx

### PR DESCRIPTION
**Pull request by OwenArnold**
Note the behaviour of `is_approx` has changed from what we had before. It now works more like `np.is_close` and would therefore need to be piped with `sc.all` to achieve the same desired legacy effect. I think the new behaviour is better consistent with `sc.equal` etc.

### CODE REVIEW

**Comment by SimonHeybrock** [comment]
>     * Type list (hard-coded), but maybe this cannot be avoided without changes to macro `INSTANTIATE_VARIABLE` - Note that `bind_data_access.h` has a method that looks to be doing similar things `static outVariant_t<View> get(View &view)`

Accept the tolerance as a variable.

File: core/include/scipp/core/element/comparison.h [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 #include "scipp/common/overloaded.h"
 #include "scipp/core/element/arg_list.h"
+#include "scipp/core/element/math.h"
```
Comment (SimonHeybrock) [inline core/include/scipp/core/element/comparison.h:11]: Where is this needed?

**Review by SimonHeybrock — APPROVED** [review]

File: core/include/scipp/core/element/comparison.h (hunk 1 above)
Comment (OwenArnold) [reply core/include/scipp/core/element/comparison.h:11]: element wise `abs`

Comment (SimonHeybrock) [reply core/include/scipp/core/element/comparison.h:11]: I don't see that? You are using `abs` which should either be `std::abs` or the `abs` for `ValueAndVariance`?

Comment (OwenArnold) [reply core/include/scipp/core/element/comparison.h]: Yes you are right can be done directly. Should be fixed now.

Comment (SimonHeybrock) [reply core/include/scipp/core/element/comparison.h]: Ah, I see now with your update, it was using the lambda to avoid the direct call/lookup.

**Review by SimonHeybrock — APPROVED** [review]

**Pull request merged**